core/src/test/scala/unit/kafka/docker/KafkaDockerWrapperTest.scala

```diff
@@ -109,25 +109,6 @@ class KafkaDockerWrapperTest {
     assertEquals(expected, actual)
   }
 
-  @Test
-  def testPrepareServerConfigsWithEmptyMountedFile(): Unit = {
```

Comment:
1. Why do we remove this test?
2. I expect we should have a test for envVars, could you help add one?

Thanks.